package.json

```diff
@@ -46,19 +46,8 @@
   "dependencies": {
     "@ohdsi/atlascharts": "^1.3.1",
     "@stomp/stompjs": "github:stomp-js/stomp-websocket",
-    "arachne-ui-components": "^1.13.44",
+    "arachne-ui-components": "^1.13.46",
     "autoprefixer": "^6.7.6",
-    "babel-core": "^6.25.0",
```

Comment:
it is build-time dependencies which also duplicated in dev-Dependencies section so I merged them there